### PR TITLE
Translating Organize Cluster Access to ID Localization

### PR DESCRIPTION
**Pull request by wahyuoi**
Addresses #15341 
/assign @irvifa

### CODE REVIEW

File: content/id/docs/concepts/configuration/organize-cluster-access-kubeconfig.md [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+
+{{% capture overview %}}
+
+Gunakan berkas kubeconfig untuk mengatur informasi mengenai kluster, pengguna, 
+_namespace_, dan mekanisme autentikasi. Perintah `kubectl` menggunakan berkas
+kubeconfig untuk mencari informasi yang dibutuhkan untuk memilih kluster dan
+berkomunikasi dengan API server dari suatu kluster.
+
+{{< note >}}
+Sebuah file yang digunakan untuk mengatur akses pada kluster disebut dengan
```
Comment (girikuncoro) [inline content/id/docs/concepts/configuration/organize-cluster-access-kubeconfig.md]:
```suggestion
Sebuah berkas yang digunakan untuk mengatur akses pada kluster disebut dengan
```

File: content/id/docs/concepts/configuration/organize-cluster-access-kubeconfig.md [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+berkas kubeconfig. Ini cara yang umum digunakan untuk mereferensikan berkas
+konfigurasi. Ini tidak berarti ada berkas dengan nama `kubeconfig`.
+{{< /note >}}
+
+Secara _default_, `kubectl` mencari file dengan nama `config` pada direktori
```
Comment (girikuncoro) [inline content/id/docs/concepts/configuration/organize-cluster-access-kubeconfig.md]:
```suggestion
Secara _default_, `kubectl` mencari berkas dengan nama `config` pada direktori
```

File: content/id/docs/concepts/configuration/organize-cluster-access-kubeconfig.md [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+
+Berikut adalah aturan yang digunakan `kubectl` ketika menggabungkan beberapa berkas
+kubeconfig:
+
+1. Jika menggunakan tanda `--kubeconfig`, maka akan menggunakan berkas yang ditentukan.
+   Tidak digabungkan. Hanya 1 tanda `--kubeconfig` yang diperbolehkan.
+
+   Sebaliknya, jika variabel _environment_ `KUBECONFIG` digunakan, maka akan menggunakan 
+   ini sebagai berkas-berkas yang akan digabungkan. Penggabungan berkas-berkas yang terdapat
+   pada variabel _environment_ `KUBECONFIG` akan menggikuti aturan sebagai berikut:
```
Comment (girikuncoro) [inline content/id/docs/concepts/configuration/organize-cluster-access-kubeconfig.md]: typo
```suggestion
   pada variabel _environment_ `KUBECONFIG` akan mengikuti aturan sebagai berikut:
```

File: content/id/docs/concepts/configuration/organize-cluster-access-kubeconfig.md [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+
+   * Mengabaikan berkas tanpa nama.
+   * Mengeluarkan pesan kesalahan untuk berkas dengan isi yang tidak dapat di-_deserialized_.
```
Comment (girikuncoro) [inline content/id/docs/concepts/configuration/organize-cluster-access-kubeconfig.md]: how about deserialisasi?
```suggestion
   * Mengeluarkan pesan kesalahan untuk berkas dengan isi yang tidak dapat dideserialisasi.
```

File: content/id/docs/concepts/configuration/organize-cluster-access-kubeconfig.md [hunk 5]
```diff
@@ -0,0 +1,165 @@
...
+   * Berkas pertama yang menentukan nilai atau _key_ pada _map_ maka akan digunakan 
+     pada _map_ tersebut.
+   * Tidak pernah mengubah nilai atau _key_ dari suatu _map_.
+     Contoh: Pertahakan konteks pada berkas pertama yang mengatur `current-context`.
```
Comment (girikuncoro) [inline content/id/docs/concepts/configuration/organize-cluster-access-kubeconfig.md]: typo
```suggestion
     Contoh: Pertahankan konteks pada berkas pertama yang mengatur `current-context`.
```

File: content/id/docs/concepts/configuration/organize-cluster-access-kubeconfig.md [hunk 6]
```diff
@@ -0,0 +1,165 @@
...
+     Contoh: Jika terdapat dua file yang menentukan nilai `red-user`, maka hanya gunakan
```
Comment (girikuncoro) [inline content/id/docs/concepts/configuration/organize-cluster-access-kubeconfig.md]:
```suggestion
     Contoh: Jika terdapat dua berkas yang menentukan nilai `red-user`, maka hanya gunakan
```

File: content/id/docs/concepts/configuration/organize-cluster-access-kubeconfig.md [hunk 7]
```diff
@@ -0,0 +1,165 @@
...
+
+1. Untuk setiap informasi yang masih belum terisi, akan menggunakan nilai `default` dan
+   kemungkinan akan meminta informasi autentikasi.
+
+## Referensi berkas
+
+Referensi _file_ dan _path_ pada berkas kubeconfig adalah bernilai relatif terhadap
+lokasi dari berkas kubeconfig.
+Referensi _file_ pada perintah adalah relatif terhadap direktori kerja saat ini.
+Dalam `$HOME/.kube/config`, _relative paths_ akan disimpan secara relatif, dan
```
Comment (girikuncoro) [inline content/id/docs/concepts/configuration/organize-cluster-access-kubeconfig.md]: singular is preferred for imported word
```suggestion
Dalam `$HOME/.kube/config`, _relative path_ akan disimpan secara relatif, dan
```

File: content/id/docs/concepts/configuration/organize-cluster-access-kubeconfig.md [hunk 8]
```diff
@@ -0,0 +1,165 @@
...
+_absolute paths_ akan disimpan secara mutlak.
```
Comment (girikuncoro) [inline content/id/docs/concepts/configuration/organize-cluster-access-kubeconfig.md]:
```suggestion
_absolute path_ akan disimpan secara mutlak.
```

File: content/id/docs/concepts/configuration/organize-cluster-access-kubeconfig.md [hunk 9]
```diff
@@ -0,0 +1,165 @@
...
+
+{{% /capture %}}
+
+
+{{% capture whatsnext %}}
+
+* [Mengature Akses Pada Beberapa Kluster](/docs/tasks/access-application-cluster/configure-access-multiple-clusters/)
```
Comment (girikuncoro) [inline content/id/docs/concepts/configuration/organize-cluster-access-kubeconfig.md]: typo
```suggestion
* [Mengatur Akses Pada Beberapa Kluster](/docs/tasks/access-application-cluster/configure-access-multiple-clusters/)
```

**Comment by phanama** [comment]
Woops, can't do it yet haha.
Let's wait for @girikuncoro or @irvifa then @wahyuoi

**Comment by girikuncoro** [comment]
thank you for your contribution! 🎉 please help us translate other pages (or help us improve the process of translation) if you have time

/lgtm
/approve

**Pull request merged**